commands/update-check.go

```diff
@@ -0,0 +1,95 @@
+package commands
+
+import (
+	"time"
+	"net/http"
+	"io/ioutil"
+	"strings"
+	"strconv"
+
+	"github.com/activecm/rita/resources"
+	"github.com/activecm/rita/config"
+	log "github.com/sirupsen/logrus"
+)
+
+// UpdateCheck Performs a check for the new version of RITA against the git repository and
+//returns a string indicating the new version if available
+func UpdateCheck( configFile string) string {
+	res := resources.InitResources( configFile )
+	delta := res.Config.S.UserConfig.UpdateCheckFrequency
+
+	if delta == 0 {
+		return ""
+	}
+
+	//Check Log for Versioning
+	m := res.MetaDB
+	t, s := m.LastCheck()
+
+	days := time.Now().Sub( t ).Hours()/24
+
+	if days > float64(delta) {
+
+		s = remoteVersion()
+
+		if s == "" {
```

Comment:
I changed `remoteVersion()` to `getRemoteVerion()` and made it return `(string error)`. Since the string is never used anyway if an error occured an error seemed more appropriate.

If you would like additional comments just let me know.